extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -61,6 +61,7 @@ import {insertAnalyticsElement} from '../../../src/extension-analytics';
 import {
   getAdSenseAmpAutoAdsExpBranch,
 } from '../../../ads/google/adsense-amp-auto-ads';
+import {validateAdSenseAdHeight} from '../../../ads/google/adsense.js'
```

Comment:
Do we need this check here as well? (the validateData check is only in the other file)

Reply:
Yes, this is necessary for the a4a path, doesn't work without it.